### DISCUSSION TITLE
Transaction stuck

### DISCUSSION THREAD

**Discussion by Grrslite** [zkSync-Community-Hub/zksync-developers#587] [General]
### Environment

Mainnet

### Provide a brief description of the functionality you're trying to implement and the issue you are running into.

Trying to bridge from ZKSync, but transaction is stuck: 0xb99db885ea43c77a7bcdfa2f8ee295794868609a85573915beb79e009151481a

**Comment by 0xsisyfos** [comment]
This space is for developers.

For questions about your transactions, please join Discord: https://join.zksync.dev/
